code/SS_Datetimezone.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Class SS_Datetimezone
+ *
+ * Adds customisable timezones to the nice method on {@link SS_Datetime}.
+ */
+class SS_Datetimezone extends SS_Datetime {
+	/**
+	* Returns the date in the raw SQL-format specific to a given timezone passed from the Member class, e.g. “2006-01-18 16:32:04”
+	*/
+	public function Nice() {
+		//instantiate new DateTime object based off received timestamp in the default timezone
+		$timestamp = new DateTime($this->value, new DateTimeZone(date_default_timezone_get()));
```

Comment:
This could be tidied up for reduced line lengths and less conditional checking by doing something like the following:

```
public function Nice() {
	$date = new DateTime($this->value);
	$member = Member::currentUser();
	if ($member && $member->exists() && $member->Timezone) {
		$date->setTimezone(new DateTimeZone($member->Timezone));
	}
	return $date->Format('d/m/Y g:ia');
}
```

The `date_default_timezone_get()` isn't necessary either, as the constructor for DateTime will default to the current timezone.